Dockerfile_race_test

```diff
@@ -19,7 +19,6 @@ RUN curl -fsSL -o /usr/local/bin/dep https://github.com/golang/dep/releases/down
 WORKDIR /go/src/github.com/tektoncd/dashboard
 COPY . .
 RUN dep ensure -vendor-only
-ENV WEB_RESOURCES_DIR=/go/src/github.com/tektoncd/dashboard/testdata/web/
```

Comment:
Not sure about this

Reply:
If you were to run the tests without this, does everything pass? Safe to remove if so

Reply:
Running locally ? This is not used in CI ?

Reply:
No impact